### ISSUE TITLE
Please remove the import Iris

### ISSUE THREAD

**Issue by dxvgef** [fasthttp-contrib/sessions#2]
errors.go file import Iris, please remove the import Iris.

**Comment by acidvertigo** [comment]
same here github.com/kataras/iris/errors package does not exists anymore

**Comment by kataras** [comment]
Yes, I converted a new go-sessions packages, which supports both net/http and fasthttp
